packages/apptive_grid_form/lib/src/apptive_grid_form.dart

```diff
@@ -653,8 +675,10 @@ class ApptiveGridFormDataState extends State<ApptiveGridFormData> {
 
   EdgeInsets get _defaultPadding => const EdgeInsets.all(8.0);
 
-  Future<void> _submitForm(ApptiveLink link, BuildContext buildContext) async {
-    if (_formKey.currentState!.validate()) {
+  /// Submits the [currentData] if not already submitting
+  Future<void> submitForm(BuildContext buildContext) async {
+    final link = _formData?.links[ApptiveLinkType.submit];
```

Comment:
Looks like we don't actually need to pass in a buildcontext here as the context of the state itself should always have access to the `ApptiveGridLocalization` so we can then even use tearoffs and in total that feels like a cleaner api as well if `submitForm` is not related to the context.

Reply:
done